[0.011s] Invoking command in '/home/ihitexe/mrm_ros_union_dir/ros_ws3/build/interface': CMAKE_PREFIX_PATH=/opt/ros/humble /usr/bin/cmake /home/ihitexe/mrm_ros_union_dir/ros_ws3/src/interface -DCMAKE_INSTALL_PREFIX=/home/ihitexe/mrm_ros_union_dir/ros_ws3/install/interface
[0.023s] -- Found ament_cmake: 1.3.10 (/opt/ros/humble/share/ament_cmake/cmake)
[0.126s] -- Found rosidl_default_generators: 1.2.0 (/opt/ros/humble/share/rosidl_default_generators/cmake)
[0.127s] -- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
[0.129s] -- Found rosidl_adapter: 3.1.5 (/opt/ros/humble/share/rosidl_adapter/cmake)
[0.149s] -- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
[0.312s] [31mCMake Error at /opt/ros/humble/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:59 (message):
[0.312s]   execute_process(/usr/bin/python3 -m rosidl_adapter --package-name interface
[0.312s]   --arguments-file
[0.312s]   /home/ihitexe/mrm_ros_union_dir/ros_ws3/build/interface/rosidl_adapter__arguments__interface.json
[0.312s]   --output-dir
[0.312s]   /home/ihitexe/mrm_ros_union_dir/ros_ws3/build/interface/rosidl_adapter/interface
[0.312s]   --output-file
[0.312s]   /home/ihitexe/mrm_ros_union_dir/ros_ws3/build/interface/rosidl_adapter/interface.idls)
[0.312s]   returned error code 1:
[0.313s] 
[0.313s]   Traceback (most recent call last):
[0.313s] 
[0.313s]     File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
[0.313s]       return _run_code(code, main_globals, None,
[0.313s]     File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
[0.313s]       exec(code, run_globals)
[0.313s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__main__.py", line 19, in <module>
[0.313s]       sys.exit(main())
[0.313s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/main.py", line 53, in main
[0.313s]       abs_idl_file = convert_to_idl(
[0.313s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__init__.py", line 19, in convert_to_idl
[0.314s]       return convert_msg_to_idl(
[0.314s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/msg/__init__.py", line 28, in convert_msg_to_idl
[0.314s]       msg = parse_message_string(package_name, input_file.stem, content)
[0.314s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 544, in parse_message_string
[0.314s]       msg = MessageSpecification(pkg_name, msg_name, fields, constants)
[0.314s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 388, in __init__
[0.314s]       self.base_type = BaseType(
[0.314s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 205, in __init__
[0.314s]       raise InvalidResourceName(
[0.314s] 
[0.314s]   rosidl_adapter.parser.InvalidResourceName: 'bill' is an invalid message
[0.315s]   name.  It should have the pattern '^[A-Z][A-Za-z0-9]*$'
[0.315s] 
[0.315s] Call Stack (most recent call first):
[0.315s]   /opt/ros/humble/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
[0.315s]   CMakeLists.txt:19 (rosidl_generate_interfaces)
[0.315s] 
[0.315s] [0m
[0.315s] -- Configuring incomplete, errors occurred!
[0.315s] See also "/home/ihitexe/mrm_ros_union_dir/ros_ws3/build/interface/CMakeFiles/CMakeOutput.log".
[0.319s] Invoked command in '/home/ihitexe/mrm_ros_union_dir/ros_ws3/build/interface' returned '1': CMAKE_PREFIX_PATH=/opt/ros/humble /usr/bin/cmake /home/ihitexe/mrm_ros_union_dir/ros_ws3/src/interface -DCMAKE_INSTALL_PREFIX=/home/ihitexe/mrm_ros_union_dir/ros_ws3/install/interface
